plugins/checkpoint_sand_blast/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-requests==2.19.1
+requests==2.26.0
```

Comment:
Is this necessary. I thought the requests is pinned in SDK. Kindly correct me if I am wrong

Reply:
We can, it would then be safe to remove any pinning of requests then